src/main/javascript/src/views/assignment/TerracottaBuilder.vue

```diff
@@ -216,6 +222,7 @@ export default {
       updateQuestion: "assessment/updateQuestion",
       deleteQuestion: "assessment/deleteQuestion",
       updateAnswer: "assessment/updateAnswer",
+      removeAssessmentQuestions: "assessment/removeAssessmentQuestions",
```

Comment:
assessment/removeAssessmentQuestions does seem to exist and isn't used in this code. Is it needed?

Reply:
After looking at the Jira issue, which says

> Use Assessment controller endpoint: PUT "/{experiment_id}/conditions/{condition_id}/treatments/{treatment_id}/assessments/{assessment_id}" with an empty questions list. JPA should do an orphan removal automatically

Maybe that was the idea with removeAssessmentQuestions, to remove these questions in bulk, which seems like a better way to do it.

Reply:
Initially that was my plan, but the endpoint didn't seem to support that, and since I saw the loop method was already being used in "handleSaveQuestions", this seemed like a decent alternative.

I've fixed (removed) the reference to removeAssessmentQuestions.